Products/ZenModel/skins/zenmodel/editSettings.pt

```diff
@@ -210,6 +210,18 @@ loader.insert();
         </td>
     </tr>
 
+    <tr>
+        <td style="color:darkred;width:275px;height:50px" class=tableheader align="left">
+            <strong>Enter current password to confirm changes</strong>
+        </td>
+        <td align="left" style="padding:4px 0 0 4px;" class="tableheader" colspan=3>
```

Comment:
Styling change here.  HTML/XML specifies that all attribute values should be quoted, which includes the `colspan` attribute.  HTML is forgiving for legacy reasons which is why it didn't blow up on you.